sleap_nn/training/model_trainer.py

```
@@ -109,75 +110,97 @@ def __init__(self, config: OmegaConf):
-                func=factory_get_chunks,
-                train_labels=train_labels,
-                val_labels=val_labels,
+    def _create_data_loaders(self):
+        """Create a DataLoader for train, validation and test sets using the data_config."""
+
+        def run_subprocess():
+            process = subprocess.Popen(
+                [
+                    "python",
+                    "-m",
+                    f"sleap_nn.training.get_bin_files",
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)